ocaml/oidc/client.nix

```diff
@@ -1,10 +1,19 @@
-{ lib, buildDunePackage, oidc, jose, uri, yojson, logs, piaf }:
+{ lib, buildDunePackage, oidc, jose, uri, yojson, logs, piaf-lwt }:
```

Comment:
Maybe time to remove this?

Reply:
I don't care. this PR fixes it, so maybe not yet?